target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-wifi.dts

```diff
@@ -0,0 +1,402 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Copyright (c) 2016, 2018 The Linux Foundation. All rights reserved.
+ * Copyright (c) 2016 Google, Inc
+ */
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	model = "Google WiFi (Gale)";
+	compatible = "google,wifi", "google,gale-v2", "qcom,ipq4019";
+
+	chosen {
+		stdout-path = &blsp1_uart1;
+	};
+
+	memory {
+		device_type = "memory";
+		reg = <0x80000000 0x20000000>; /* 512MB */
+	};
+
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+
+		rsvd@87000000 {
+			/* Reserved for other subsystem */
```

Comment:
What is this for?

Reply:
Good question; I forgot I just borrowed that from the product DTS as-is.
Judging by some other context (the bootloader actually adds in some additional reserved-memory nodes of its own), I believe it's a conservative range carved out for some ATF/TrustZone usage. And in fact, Linux complains about some overlap at bootup, because the bootloader-provided regions sit right in the middle of this one.

I suspect it's OK to simply drop this node and defer to the bootloader instead. I plan to do that for the next revision.